R/get_event_rsvps.R

```diff
@@ -0,0 +1,41 @@
+#' Get the RSVPs for a specified event
+#'
+#' @template urlname
+#' @param event_id Character. The id of the event. Event ids can be obtained
+#'   using [get_events()] or by looking at the event page URL.
+#' @template api_key
+#'
+#' @return A tibble with the following columns:
+#'    * member_id
+#'    * member_name
+#'    * member_is_host
+#'    * response
+#'    * guests
+#'    * created
+#'    * updated
+#'    * resource
+#' @references
+#' \url{https://www.meetup.com/meetup_api/docs/:urlname/events/:event_id/rsvps/#list}
+#' @examples
+#' \dontrun{
+#' urlname <- "rladies-nashville"
+#' upcoming_events <- get_events(urlname = urlname,
+#'                       event_status = "upcoming")
```

Comment:
This should be `event_status = "past"` to get past events (I assume that's what you want since there will always be past events and the next line says: `#first event for this group`.  I will make the update on master after I merge.